mobile-browser-based-version/src/helpers/training/training_manager.js

```diff
@@ -144,12 +180,32 @@ export class TrainingManager {
   async _trainLocally() {
     const info = this.task.trainingInformation;
 
-    await this.model.fit(this.data, this.labels, {
-      initialEpoch: this.model.getUserDefinedMetadata().epoch,
-      epochs: info.epochs ?? MANY_EPOCHS,
-      batchSize: info.batchSize,
-      validationSplit: info.validationSplit,
-      shuffle: true,
+    var dataSize = -1;
+
+    if (info.resize) {
```

Comment:
What happens if `info.resize` is not set? Will `dataSize` remain equal to -1 and "fast training mode" be the default?